apps/web/pages/settings/organizations/appearance.tsx

```diff
@@ -0,0 +1,9 @@
+import OrgAppearanceView from "@calcom/features/ee/organizations/pages/settings/appearance";
+
+import type { CalPageWrapper } from "@components/PageWrapper";
+import PageWrapper from "@components/PageWrapper";
+
+const Page = OrgAppearanceView as CalPageWrapper;
```

Comment:
Why do we need type casting here?

Reply:
Not sure - types seemed to break without it and it appears to be in other pages also 🤔